kura/examples/test/org.eclipse.kura.example.wire.bool.multiport.provider.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0
```

Comment:
Similar comments as for the component under test, including renaming the project, packages, DS etc

Reply:
noted (I think).